.golangci-kal.yml

```
@@ -42,6 +43,18 @@ linters:
             disable:
             - "*" # We will manually enable new linters after understanding the impact. Disable all by default.
           lintersConfig:
+            conflictingmarkers:
+              conflicts:
+              - name: "optional_vs_required"
+                sets:
+                  - ["optional", "+kubebuilder:validation:Optional", "+k8s:validation:optional"]
```

There's no `validation` in the middle
```suggestion
                  - ["optional", "+kubebuilder:validation:Optional", "+k8s:optional"]
```